torch_xla/test/test_utils.py

```diff
@@ -10,19 +12,33 @@ def _get_device_spec(device):
   return str(device) if ordinal < 0 else '{}/{}'.format(device, ordinal)
 
 
-def add_scalar_to_summary(summary_writer, metric_name, metric_value,
-                          global_step):
-  """Add a scalar data point to a SummaryWriter.
+def write_to_summary(summary_writer, global_step, dict_to_write={},
+                     write_xla_metrics=False):
+  """Writes scalars to a SummaryWriter. Optionally writes XLA perf metrics.
 
   Args:
     summary_writer: instance of Tensorboard SummaryWriter or
         None. If None, no summary files will be written.
-    metric_name: Str. Name of the metric to write.
-    metric_value: Float. Value of the metric.
-    global_step: Int. The global step value for this data point.
+    global_step: Int. The global step value for these data points.
```

Comment:
This is not the syntax we use. PTAL at examples in xla_model.py

Reply:
Fixed. I'll fix the other methods in this file in a separate cleanup PR

Reply:
Filed a bug to remember to do that